### PR TITLE
Ignoring legacy engagement shared items during serialization.

### PR DESCRIPTION
**Pull request by eschlange**
[#118937543]

### CODE REVIEW

File: spec/models/social_networking/serializers/shared_item_serializer_spec.rb [hunk 1]
```diff
@@ -4,50 +4,68 @@ module SocialNetworking
...
-          .to_serialized
-      end
 
       describe ".to_serialized" do
-        before do
-          allow(thought).to receive(:participant) { participant }
-          allow(thought).to receive(:to_serialized)
-          allow(thought).to receive(:description) { "" }
+        context "a valid shared item type" do
+          let(:thought) { double("thought", participant_id: 1) }
```
Comment (michaelwehrley) [inline spec/models/social_networking/serializers/shared_item_serializer_spec.rb]: I would recommend `instance_double(Thought)` if possible.

**Comment by michaelwehrley** [comment]
Looks good.  Only one comment.  So the `item` that was related to the `Engagement` class?  Thanks.

File: spec/models/social_networking/serializers/shared_item_serializer_spec.rb (hunk 1 above)
Comment (eschlange) [inline spec/models/social_networking/serializers/shared_item_serializer_spec.rb]: Social networking does not have access to the Thought class, so I don't think this is possible.

**Comment by eschlange** [comment]
@michaelwehrley after speaking with Eric, I'm going to roll back this change and remove engagement data from the shared items table. This would alleviate errors swallowed from future changes.

**Pull request closed without merging**